save-backend/src/test/kotlin/com/saveourtool/save/backend/S11nTestConfig.kt

```diff
@@ -0,0 +1,88 @@
+package com.saveourtool.save.backend
+
+import org.reactivestreams.Publisher
+import org.springframework.boot.test.context.TestConfiguration
+import org.springframework.boot.test.web.reactive.server.WebTestClientBuilderCustomizer
+import org.springframework.context.annotation.Bean
+import org.springframework.core.ResolvableType
+import org.springframework.core.codec.AbstractDecoder
+import org.springframework.core.codec.AbstractEncoder
+import org.springframework.core.io.buffer.DataBuffer
+import org.springframework.core.io.buffer.DataBufferFactory
+import org.springframework.http.codec.CodecConfigurer.DefaultCodecs
+import org.springframework.http.codec.ServerCodecConfigurer
+import org.springframework.util.MimeType
+import org.springframework.web.reactive.config.WebFluxConfigurer
+import reactor.core.publisher.Flux
+
+/**
+ * Extra S11N configuration used in tests (disables `kotlinx.serialization`
```

Comment:
Btw, why `test` has such problem, but `main` doesn't?

Reply:
> Btw, why `test` has such problem, but `main` doesn't?

Because `kotlinx.serialization` is on the test classpath now &mdash; see [`save-backend/build.gradle.kts`](https://github.com/saveourtool/save-cloud/pull/1678/files#diff-d5a88653576a1c8fd2e02a0472a9a6da5aa555723b4dadae8fb989b9e1104075R88).